cypress/utils/apps/query_enhancements/commands.js

```diff
@@ -101,7 +101,7 @@ Cypress.Commands.add('deleteDataSourceByName', (dataSourceName) => {
   cy.visit('app/dataSources');
 
   // Find the anchor text corresponding to specified dataSource
-  cy.get('a').contains(dataSourceName).click();
+  cy.get('a').contains(dataSourceName).click({ force: true });
```

Comment:
sorry one more force

Reply:
did you push that change? i don't see it here :D

Reply:
I did something worse than that. I got confused with the deleteAllDataSources command and removed the force from that instead 🤦. I've retested, restored the old force and removed the one here.